docs/clustering/multi-node-setup.rst

```diff
@@ -461,8 +461,8 @@ Edit the `transport.tcp.port`_ setting in your `configuration`_ file:
     `More information about port settings`_
```

Comment:
This section seems broken to me.

Reply:
@mkleen oh you're totally right. pushed a fixup. thx!

Reply:
<img width="1260" alt="Screenshot 2020-12-18 at 14 47 59" src="https://user-images.githubusercontent.com/338234/102621643-28041280-4140-11eb-8c16-5b00c7294578.png">

Reply:
This is what i meant with broken section.

Reply:
@mkleen I think this must be an issue with your local dev setup. can you try running `make reset` inside `docs/` and then try again?

you can view the correct output on RTD here:

https://crate-howtos--263.org.readthedocs.build/en/263/clustering/multi-node-setup.html

this (the RTD build) looks fine to me

Reply:
oh you are right, sorry for that.